src/rt/typeinfo/ti_ucent.d

```diff
@@ -30,7 +28,7 @@ class TypeInfo_zk : TypeInfo
 
     override size_t getHash(scope const void* p)
     {
-        return rt.util.hash.hashOf(p[0 .. ucent.sizeof], 0);
+        return hashOf(p[0 .. ucent.sizeof], 0);
```

Comment:
To match what's in `core.internal.hash` you should instead write:
```d
return hashOf(*cast(const ucent*) p);
```

Reply:
done